cdist/conf/type/__jail_freebsd10/gencode-remote

```diff
@@ -130,11 +130,11 @@ deleteJail() {
       fi
 EOF
 # Remove the jail's rw mountpoints
-   echo "rm -rf \"${jaildir}/rw/${name}\""
+   echo "rm -rf '${jaildir}/rw/${name}'"
 # Remove the jail directory
-   echo "rm -rf \"${jaildir}/${name}\""
+   echo "rm -rf '${jaildir}/${name}'"
 # Remove the jail's fstab
-   echo "rm -f \"/etc/fstab.${name}\""
+   echo "rm -f '/etc/fstab.${name}'"
 # Remove jail entry from jail.conf
    cat <<EOF
       sed -i .bak -E -e "/^${name} {\$/,/^}\\\$/d" /etc/jail.conf
```

Comment:
`sed -i` is not defined by POSIX (double-checked ;) http://pubs.opengroup.org/onlinepubs/9699919799/utilities/sed.html

The POSIX-compliant way to do inlines is to copy the file and sed it back to its original location.